File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/ord/cfg/cirulecfg/d/RuleCategory.java
package iih.ci.ord.cfg.cirulecfg.d;

import xap.sys.appfw.orm.desc.ent.dataobject.DmEnumDesc;

public interface RuleCategory {

    @DmEnumDesc(name="校验规则",description="校验规则")
	public static final Integer VALIDATOR=1; //校验规则
    @DmEnumDesc(name="业务逻辑规则",description="业务逻辑规则")
	public static final Integer BIZ_LOGIC=2; //业务逻辑规则
}	

File: client/iih_stub/iih.ci_stub/iih.ci.ord/cfg/cirulecfg/d/RuleCategory.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.ComponentModel;

namespace iih.ci.ord.cfg.cirulecfg.d
{
   public enum RuleCategory {
	/// <summary>
	/// 校验规则
	/// </summary>
    [Description("校验规则")]
    VALIDATOR=1,
	/// <summary>
	/// 业务逻辑规则
	/// </summary>
    [Description("业务逻辑规则")]
    BIZ_LOGIC=2,
   }
}
